programs/survival/portalorient.sc

```diff
@@ -0,0 +1,186 @@
+__config() ->
+(
+	m(
+		l('scope','player'),
+		l('stay_loaded',true),
+		
+		l('commands',
+			m(
+				l('<mode>','__change')
+			)
+		),
+		l('arguments',
+			m(
+				l('mode',
+					m(
+						l('type','term'),
+						l('options',
+							l('off','air','solid')
+						),
+						l('suggest',
+							l('off','air','solid')
+						)
+					)
+				)
+			)
+		)
+	)
+);

```

Comment:
Think about changing this for the modern syntax that uses `[ ]` and `{ }` for lsits and maps respectively.

Reply:
I think I got them all.